tests/utils/test_samplesheet.py

```diff
@@ -1663,9 +1663,13 @@ def test_x_validate_multiple_index(
 
 
 def test_hiseqx_samplesheet_wrong_columns(hiseqx_samplesheet_wrong_columns_path: Path):
-    sample_sheet = Samplesheet(hiseqx_samplesheet_wrong_columns_path.as_posix())
+    """Test validation of correct columns in a hiseqx sample sheet"""
 
+    # GIVEN a hiseqx sample sheet with incorrect headers
+    sample_sheet = Samplesheet(hiseqx_samplesheet_wrong_columns_path.as_posix())
+    # WHEN said samplesheet is validated
     with pytest.raises(SampleSheetValidationException):
+        # THEN appropriate exception should be raised
```

Comment:
Can you move the `# THEN` to the bottom and assert the error message in the `# THEN section`?